### PR TITLE
Week 1 homework

### PR DESCRIPTION
**Pull request by alexeygrigorev**

### CODE REVIEW

File: week_1_basics_n_setup/homework.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+
+Download this data and put it to Postgres
+
+## Question 3. Count records 
+
+How many taxi trips were there on January 15?
+
+## Question 4. Average
+
+Find the largest tip for each day.
```
Comment (Victoriapm) [inline week_1_basics_n_setup/homework.md:37]: I think here  and the next line you meant trip

File: week_1_basics_n_setup/homework.md [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+Install Google Cloud SDK. What's the version you have? 
+
+To get the version, run `gcloud --version`
+
+## Question 2. Terraform 
+
+Install Terraform. What's the version you have?
+
+Run `terraform -version` to get it
+
```
Comment (sejalv) [inline week_1_basics_n_setup/homework.md:17]: Also, output of `terraform apply` to check if their GCP infra was created successfully.

File: week_1_basics_n_setup/homework.md (hunk 1 above)
Comment (alexeygrigorev) [reply week_1_basics_n_setup/homework.md:37]: Actually I did not!

Comment (alexeygrigorev) [reply week_1_basics_n_setup/homework.md:37]: I was referring to `tip_amount` - made it clear that it's not a typo. Thanks!

**Pull request merged**